routes/users.js

```diff
@@ -0,0 +1,57 @@
+const express = require("express");
+const router = express.Router();
+const createError = require("http-errors");
+const admin = require("firebase-admin");
+
+const User = require("../models/User");
+
+const MESSAGE = require("../constants/messages");
+const ERROR = require("../constants/error");
+
+router.get("/check_member", async (req, res, next) => {
+  const { EXPIRED_TOKEN, UNEXPECTED_ERROR, OK } = MESSAGE;
+  const { AUTH_ID_TOKEN_REVOKED, AUTH_ID_TOKEN_EXPIRED } = ERROR;
+  const { idToken } = req.body;
+
+  try {
+    const decodeToken = await admin
+      .auth()
+      .verifyIdToken(idToken, false);
+
+    const { email } = decodeToken;
+
+    const user = await User.findOne({ email });
+
+    const hasUserData = user === null ? false : true;
```

Comment:
삼항연산자로 bool 값을 반환하고 계신데, 다음과 같이 작성하면 좋을 것 같습니다!
```suggestion
    const hasUserData = !!user;
```

Reply:
`const hasUserData = user !== null;`
이러한 방식은 어떨까요?

Reply:
토의한 것 과 같이 이번 경우에는 
`const hasUserData = user !== null;`
작성하겠습니다!